### PR TITLE
fix(): better dashboard responsiveness

### LINKED ISSUE #1563: Home Dashboard: unresponsive display of tiles displaying the number of Successful, Failed and Running Executions
### Expected Behavior

_No response_

### Actual Behaviour

Display is not responsive for digits in dashboard
image
image

### Steps To Reproduce

_No response_

### Environment Information

- Kestra Version:
- Operating System (OS / Docker / Kubernetes):
- Java Version (If not docker):

### Example flow

_No response_

### PR DESCRIPTION
**Pull request by Skraye**
Did reduce the police size as with the breaking text is was rendering really weird and happening with not this high number.

close #1563

### CODE REVIEW

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]
I don't think this is the proper way to  handle it :/
See this screen with not that small width and it still occurs
image

I suggest to use `flex-wrap: wrap` + remove `margins` in favor of `gap` property

**Comment by Skraye** [comment]
Added flex-wrap to avoid this

**Comment by tchiotludo** [comment]
already done here : https://github.com/kestra-io/kestra/pull/2184

File: ui/src/components/home/HomeSummaryStatusLabel.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
     <div class="list">
         <template v-for="[status, count] of sorted">
-            <div v-if="count > 0" :key="status" class="d-flex w-100 justify-content-between mb-2">
-                <div class="me-4 icon">
+            <div v-if="count > 0" :key="status" class="d-flex justify-content-around mb-2 flex-wrap">
```
Comment (brian-mulier-p) [inline ui/src/components/home/HomeSummaryStatusLabel.vue:4]: Please remove any mb / me & use 
```
column-gap: calc(var(--spacer) * 1.5);
row-gap: calc(var(--spacer) / 2);
```

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]

**Review by brian-mulier-p — APPROVED** [review]
LGTM

**Pull request merged**